integrations/src/main/scala/com/quantemplate/integrations/gmaps/GeocodingService.scala

```diff
@@ -0,0 +1,92 @@
+package com.quantemplate.integrations.gmaps
+
+import com.google.maps.{GeocodingApi, GeoApiContext, PendingResult}
+import com.google.maps.errors.ZeroResultsException
+import com.google.maps.model.GeocodingResult
+import scala.concurrent.{Future, Promise}
+import scala.concurrent.ExecutionContext
+import com.google.maps.model.AddressComponent
+import com.google.maps.model.AddressComponentType
+import cats.syntax.traverse.given
+import cats.syntax.option.given
+
+import com.quantemplate.integrations.common.{Config, View, Xlsx}
+
+class GeocodingService()(using conf: Config.GoogleMaps):
+  import GeocodingService.*
+  
+  private lazy val context = GeoApiContext.Builder()
+    .apiKey(conf.apiKey)
+    .build()
+
+
+  def getGeocodedRows(addresses: Vector[String])(using ExecutionContext) =
```

Comment:
missing return type

Reply:
btw, it would be great to get compilation warning on missing return type + fail on warnings

Reply:
Yes! I just have to read up on which `scalac` options are recommended for Scala 3.